ArduCopter/AP_Arming.cpp

```diff
@@ -574,7 +574,8 @@ bool AP_Arming_Copter::mandatory_gps_checks(bool display_failure)
     }
 
     // check home and EKF origin are not too far
```

Comment:
we should change this comment to "check current position and home are not too far from EKF origin"